src/ui/view/dataverse/component/generalMetadata/dateFromTo.tsx

```diff
@@ -0,0 +1,76 @@
+import { useTranslation } from 'react-i18next'
+import * as E from 'fp-ts/Either'
+import { pipe } from 'fp-ts/function'
+import {
+  convertToLocalizedDateIfISODateTime,
+  validateISODateRange
+} from '@/util/isoDateTime/isoDateTime'
+import { activeLanguageWithDefault } from '@/ui/languages/languages'
+
+const FormatDateSpan = ({
+  localizedDate,
+  label
+}: {
+  label: 'from' | 'to'
+  localizedDate: string
+}): JSX.Element => {
+  const { t } = useTranslation('common')
+  return (
+    <span>
+      <b>
```

Comment:
Usage of `<b>` is discouraged now (it was used to render bold text before HTML 4 arrived), it's better to use css for styling.

Reply:
Good point! I will style it with a class.

Reply:
@lolottetheclash  On the mockup From and To have the same font weight as the date. I ended up removing the <b> tag.